rpcprobe/main.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2025 Flant JSC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	connlib "github.com/kubernetes-csi/csi-lib-utils/connection"
+	"github.com/kubernetes-csi/csi-lib-utils/rpc"
+)
+
+type Probe struct{}
+
+func main() {
+	if len(os.Args) != 2 {
+		fmt.Println("Usage: program <unix_socket_path>")
+		fmt.Println("Example: program /tmp/rpc.sock")
```

Comment:
program name is "rpcprobe" - use it in help message
```suggestion
		fmt.Println("Usage: rpcprobe <unix_socket_path>")
```